config/groovy/util.groovy

```diff
@@ -265,48 +266,49 @@ private void printListItemsCondensed(String[] items) {
-    println ""
-    println "Example: 'groovyw module recurse GooeysQuests Sample' - would retrieve those modules plus their dependencies as source"
-    println "Example: 'groovyw lib list' - would list library projects compatible with being embedded in a Terasology workspace"
-    println "Example: 'groovyw module createDependencyDotFile JoshariasSurvival' - would create a dot file with JS' dependencies and all their dependencies - if locally available"
-    println ""
-    println "*NOTE*: Item names are case sensitive. If you add items then `gradlew idea` or similar may be needed to refresh your IDE"
-    println ""
-    println "If you omit further arguments beyond the sub-command you'll be prompted for details"
-    println ""
-    println "For advanced usage see project documentation. For instance you can provide an alternative GitHub home"
-    println "A gradle.properties file (one exists under '/templates' in an engine workspace) can provide such overrides"
-    println ""
+    println("""
+        Utility script for interacting with Terasology. General syntax:
+        groovyw (type) (sub-command)
+        - 'type' may be module,meta,lib or facade.
+        
+        Available sub-commands:
+        - 'get' - retrieves one or more items in source form (separate with spaces)
+        - 'get-all' - retrieves all modules that can be found on the configured remote locations
+        - 'recurse' - retrieves the given item(s) *and* their dependencies in source form (really only for modules)
+        - 'list' - lists items that are available for download or downloaded already.
+        - 'create' - creates a new item of the given type.
+        - 'update' - updates an item (git pulls latest from current origin, if workspace is clean
+        - 'update-all' - updates all local items of the given type.
+        - 'add-remote (item) (name)' - adds a remote (name) to (item) with the default URL.
+        - 'add-remote (item) (name) (URL)' - adds a remote with the given URL
+        - 'list-remotes (item)' - lists all remotes for (item) 
+        - 'refresh' - replaces the Gradle build file for all items of the given type from the latest template
+        - 'createDependencyDotFile' - creates a dot file recursively listing dependencies of given locally available module, can be visualized with e.g. graphviz
+        
+        Available flags:
+        '-remote [someRemote]' to clone from an alternative remote, also adding the upstream org (like MovingBlocks) repo as 'origin'
+            Note: 'get' + 'recurse' only. This will override an alternativeGithubHome set via gradle.properties.
+        '-simple-list-format' to print one item per row for the 'list' sub-command, even for large numbers of items
+        '-condensed-list-format' to group items by starting letter for the 'list' sub-command (default with many items)
+        '-skip-recently-updated' (Only for update-all) to skip updating modules that have already been updated within 10 minutes
```

Comment:
This is the new line, The reset is just removing the million `println` statements to replace with a multiline  string